tests/benchdnn/inputs/concat/test_concat_ip

```diff
@@ -0,0 +1,290 @@
+4x34x8:4x32x8
+4x35x8:4x32x8
+4x36x8:4x32x8
+4x129x8:4x129x8
+
+# all types b4
+--dtag=aBc4b
+--sdt=f32,f16,bf16,s8
+--ddt=f32,f16,bf16,s8
+--stag=aBc4b:aBc4b
+--axis=1
+4x33x8:4x32x8
+4x129x8:4x129x8
+
+# increasing inner dim
+--dtag=aBc16b
+--sdt=f32,f16,bf16,s8
+--ddt=f32,f16,bf16,s8
+--stag=aBc16b:aBc16b
+--axis=1
+4x33x8:4x32x8
+4x33x9:4x32x9
+4x33x10:4x32x10
+4x33x11:4x32x11
+4x33x12:4x32x12
+4x33x13:4x32x13
+4x33x14:4x32x14
+4x33x15:4x32x15
+4x33x16:4x32x16
+4x33x17:4x32x17
+
+4x129x8:4x129x8
+4x129x9:4x129x9
+4x129x10:4x129x10
+4x129x11:4x129x11
+4x129x12:4x129x12
+4x129x13:4x129x13
+4x129x14:4x129x14
+4x129x15:4x129x15
+4x129x16:4x129x16
+4x129x17:4x129x17
+
+# increasing inner dim
+--dtag=aBc32b
+--sdt=f32,f16,bf16,s8,u8
+--ddt=f32,f16,bf16,s8,u8
+--stag=aBc32b:aBc32b
+--axis=1
+4x33x17:4x32x17
+4x33x18:4x32x18
+4x33x19:4x32x19
+4x33x20:4x32x20
+4x33x21:4x32x21
+4x33x22:4x32x22
+4x33x23:4x32x23
+4x33x24:4x32x24
+4x33x25:4x32x25
+4x33x26:4x32x26
+4x33x27:4x32x27
+4x33x28:4x32x28
+4x33x29:4x32x29
+4x33x30:4x32x30
+4x33x31:4x32x31
+4x33x32:4x32x32
+4x33x33:4x32x33
+
+4x129x17:4x129x17
+4x129x18:4x129x18
+4x129x19:4x129x19
+4x129x20:4x129x20
+4x129x21:4x129x21
+4x129x22:4x129x22
+4x129x23:4x129x23
+4x129x24:4x129x24
+4x129x25:4x129x25
+4x129x26:4x129x26
+4x129x27:4x129x27
+4x129x28:4x129x28
+4x129x29:4x129x29
+4x129x30:4x129x30
+4x129x31:4x129x31
+4x129x32:4x129x32
+4x129x33:4x129x33
+
+--dtag=aBc16b
+--sdt=f32,f16,bf16,s8
+--ddt=f32,f16,bf16,s8
+--stag=aBc16b:aBc16b
+--axis=1
+4x33x8:4x32x8
+4x33x9:4x32x9
+4x33x10:4x32x10
+4x33x11:4x32x11
+4x33x12:4x32x12
+4x33x13:4x32x13
+4x33x14:4x32x14
+4x33x15:4x32x15
+4x33x16:4x32x16
+4x33x17:4x32x17
+
+4x129x8:4x129x8
+4x129x9:4x129x9
+4x129x10:4x129x10
+4x129x11:4x129x11
+4x129x12:4x129x12
+4x129x13:4x129x13
+4x129x14:4x129x14
+4x129x15:4x129x15
+4x129x16:4x129x16
+4x129x17:4x129x17
+
+--dtag=aBc8b
+--sdt=f32,f16,bf16,s8
+--ddt=f32,f16,bf16,s8
+--stag=aBc8b:aBc8b
+--axis=1
+4x33x12:4x32x12
+4x33x13:4x32x13
+4x33x14:4x32x14
+4x33x15:4x32x15
+4x33x16:4x32x16
+4x33x17:4x32x17
+
+4x129x12:4x129x12
+4x129x13:4x129x13
+4x129x14:4x129x14
+4x129x15:4x129x15
+4x129x16:4x129x16
+4x129x17:4x129x17
+
+--dtag=aBc4b
+--sdt=f32,f16,bf16,s8
+--ddt=f32,f16,bf16,s8
+--stag=aBc4b:aBc4b
+--axis=1
+4x33x14:4x32x14
+4x33x15:4x32x15
+4x33x16:4x32x16
+4x33x17:4x32x17
+
+4x129x14:4x129x14
+4x129x15:4x129x15
+4x129x16:4x129x16
+4x129x17:4x129x17
+
+# tests w/small concat
+--reset
+--dtag=aBc4b
+--stag=aBc4b:aBc4b
+--axis=1
+# src0 < elems_per_sg
+4x2x8:4x2x8
+# sg_size > boundary (no trailing blocks)
+4x33x8:4x4x8
+# boundary_blocks only
+4x3x32:4x3x32
+
+# large internal padding concat for idx_t=ulong
+--reset
+--dtag=aBc32b
+--stag=aBc32b:aBc32b
+--axis=1
+--sdt=bf16 --ddt=bf16 4x337810x992:4x17x992
+
+# test varying internal padding tags/formats
+--reset
+--sdt=u8
+--ddt=u8
+--axis=1
+--stag=aBcd32b:aBcd32b
+--dtag=aBcd32b
+4x33x8x8:4x32x8x8
+
+--stag=ABcd32a32b:ABcd32a32b
+--dtag=ABcd32a32b
+4x33x8x8:4x32x8x8
```

Comment:
That's too much to validate a pretty narrow new use case.
Please reduce and try to incorporate into existing files.
And this file won't be picked for GPU validation besides that.

Additionally, would be nice to understand where the blocked layout comes from because, AFAIK, activations must move to nxc format for whole model since Xe2.

Reply:
Reduced the number of tests and incorporated into gpu validation. Do let me know if there are still too many. 

There has been some back and forth on the necessity of block support. There were talks it'll still be needed with future platforms but don't recall the details for which customer. Would like to close this narrow case performance gap regardless and will revisit with more improvements(ex: 3+ inputs, large block sizes w/small simd, flexible conf, etc) if there is demand again.

Reply:
OpenVino will continue to support blocked layouts for the near future. After some discussion with @karturov  we've determined this PR introduces a sufficient number of model level improvements (including latest platforms) to warrant merging. Details in perf-report or grafana.